jobs/datadog-cc/templates/process.yaml.erb

```diff
@@ -0,0 +1,13 @@
+init_config:
+
+instances:
+  - name: cc api master
+    search_string: ['/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/bin/cloud_controller']
+    exact_match: false
+<% if p('cc.jobs.local.number_of_workers').to_i > 0 %>
+  - name: cc api worker
+    search_string: ['cc_api_worker.api']
+    exact_match: false
+    thresholds:
+      critical: [<%= p('cc.jobs.local.number_of_workers') %>, <%= p('cc.jobs.local.number_of_workers') %>]
```

Comment:
Web UI in datadog says 'Trigger the alert after selected consecutive failures:' - and it lists numbers for warn/critical.

I don't think this is related to number of processes. E.g. when you should have 3 worker processes and 2 are down, process.up would still report OK. Thus you don't need to get thresholds from number of expected processes. And if you want to monitor for number of processes, you should do it in some other way...